### ISSUE TITLE
Accessibility: Focused menu items should be highlighted

### ISSUE THREAD

**Issue by mnapoli** [piwik/piwik#6997]
To make the menus more accessible, the currently focused menu item should be highlighted.

Example of the Github menu which does this:

![capture d ecran 2015-01-14 a 10 42 25](https://cloud.githubusercontent.com/assets/720328/5729818/3899ebd6-9bda-11e4-8a11-14d48a028379.png)

**Comment by mnapoli** [comment]
Done in #7035
